src/components/PluginListItem.js

```diff
@@ -1,32 +0,0 @@
-import React from 'react';
```

Comment:
Was this file deleted intentionally?

Reply:
Yep intentional in the first commit. I think this was added in the idea that it could be reusued, but I _think_ the list item implementation has changed underneath now and its no longer usable.

Reply:
Ah, yeah, I remember coming to a similar conclusion when building out some of our local plugins.